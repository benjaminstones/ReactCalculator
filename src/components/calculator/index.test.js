import React from 'react';
import { shallow } from 'enzyme';
import '../../setupTests'
import Calculator from './index.js'

let wrapper = shallow(<Calculator />);

describe('Calculator', () => {
	it('should render the Calculator Component correctly', () => {
		expect(wrapper).toMatchSnapshot();
	});
});