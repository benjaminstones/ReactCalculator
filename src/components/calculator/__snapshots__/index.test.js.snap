// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator should render the Calculator Component correctly 1`] = `
<styled.div>
  <Display
    expression="0"
    total="0"
  />
  <ButtonContainer
    clickHandler={[Function]}
  />
</styled.div>
`;
